<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QMainWindow" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>621</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fNameLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lNameLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>340</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>390</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmPasswordLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>470</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="passwordChkBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>450</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
